doc/vic20.sgml

```diff
@@ -219,8 +219,7 @@ The program return code (low byte) is passed back to BASIC by use of the
 
 <sect1>Using extended memory<p>
 
-The extended memory at $A000 may be added to the heap by using the following
-code:
+BLK5 memory may be added to the heap by using the following code:
```

Comment:
@silverdr: I initially thought the same way too but the $A000 appears right below in the source code so one can easily make the connection between the two terms "$A000" and "BLK5". And Googling them together (even without "VIC20") work just fine. So from that perspective I considered polluks' change as an improvement even for non VIC20 experts (like i.e. me).